### DISCUSSION TITLE
What is better name to replace `asTableEx`

### DISCUSSION THREAD

**Discussion by babyfish-ct** [babyfish-ct/jimmer#790] [General]
Some users reported that the name `asTableEx` is vague and cannot alert new users, and suggested changing the name. Please tell us what you think the ideal name is.

**Comment by ArgonarioD** [comment]
`forToManyJoining`

**Comment by pot-mot** [comment]
enableCollectionJoin

**Reply by ArgonarioD** [reply]
use `Joining` or `Joins`?

**Reply by pot-mot** [reply]
Maybe just `Join`?

**Comment by babyfish-ct** [comment]
asCollectionJoinable

**Comment by wzszsw** [comment]
enableToManyJoin

**Comment by TokgoRonin** [comment]
toManyJoinable, toManyRelation

**Comment by luqiuzhi** [comment]
asToManyJoin

**Comment by belovaf** [comment]
asMultipliable()

**Comment by Enaium** [comment]
`references` or `refers`

**Reply by luqiuzhi** [reply]
good，shorter is better😏

**Reply by MistSeen** [reply]
subQuery or subWhere
